Cargo.toml

```diff
@@ -23,7 +23,7 @@ path = "src/lib.rs"
 [dependencies]
 log = "0.4.14"
 bitflags = "1.2.1"
-thiserror = "1.0.23"
+thiserror = "1.0"
```

Comment:
A newer version should be fine, but I'm wary of building against an older version than we expect. Why do you need to change this dependency?

Reply:
On both of these, I was seeing collisions between btleplug and other libraries I write farther up the stack. I suppose I could specify versions at the top level in that case? Is there a standard for this in the community?